app/controllers/search_controller.rb

```diff
@@ -9,84 +9,80 @@ def show
   end
 
   def slack
-    text = params[:text].downcase
-    channel = params[:channel_name]
-    username = params[:user_name]
-    domain = params[:team_domain]
-
-    @team = Team.find_by_domain domain
-    return render json: 'Team not found' unless @team
-    if !(text =~ /^(upvote|downvote)$/).nil?
-      vote text, domain, channel, username
-    else
-      found = find_gifs_for text
-      return no_gifs unless found
-      post_gif_to_slack(found, text, channel, username)
-    end
+    return render json: 'Team not found' unless team
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.